### PR TITLE
Release notes 7.5.0 cleanup

### PR DESCRIPTION
**Pull request by karenzone**
[edit]

Merge target branches: 7.5 and 7.x

### CODE REVIEW

**Comment by karenzone** [comment]
TODO:  Fetch release notes for 7.4.x branches

File: docs/static/releasenotes.asciidoc [hunk 1]
```diff
@@ -18,6 +19,54 @@ This section summarizes the changes in the following releases:
 * <<logstash-7-0-0-alpha2,Logstash 7.0.0-alpha2>>
 * <<logstash-7-0-0-alpha1,Logstash 7.0.0-alpha1>>
 
+[[logstash-7-5-0]]
+=== Logstash 7.5.0 Release Notes
+
+* Give multiple pipelines all the settings https://github.com/elastic/logstash/pull/11076[#11076]
+* Support substitutions in pipelines.yml file https://github.com/elastic/logstash/pull/11081[#11081]
+* Provide feature/log appender per pipeline https://github.com/elastic/logstash/pull/11108[#11108]
```
Comment (jsvd) [inline docs/static/releasenotes.asciidoc]:
```suggestion
* Provide log appender per pipeline https://github.com/elastic/logstash/pull/11108[#11108]
```

File: docs/static/releasenotes.asciidoc [hunk 2]
```diff
@@ -18,6 +19,54 @@ This section summarizes the changes in the following releases:
...
+* Use 2048 bits key in OpenSSL socket specs https://github.com/elastic/logstash/pull/11115[#11115]
+* Add origins of pipeline configurations https://github.com/elastic/logstash/pull/11123[#11123]
+* Update .ruby-version to jruby-9.2.8.0 https://github.com/elastic/logstash/pull/11198[#11198]
```
Comment (jsvd) [inline docs/static/releasenotes.asciidoc]: We can remove this one. It's a tweak in the environment setup for testing, not relevant for the release.

```suggestion
```

File: docs/static/releasenotes.asciidoc [hunk 3]
```diff
@@ -18,6 +19,54 @@ This section summarizes the changes in the following releases:
...
+* Remove 10k character truncation from log4j2.properties https://github.com/elastic/logstash/pull/11206[#11206]
+* Remove from system properties if Java runtime is less than 11 https://github.com/elastic/logstash/pull/11225[#11225] to
+fix https://github.com/elastic/logstash/issues/11221[#11221]
+* [DOC] Add metricbeat as a monitoring option https://github.com/elastic/logstash/issues/11190[#11190]
+
+TBD
+
+2804651a7 - improve reliability of webserver port binding (4 weeks ago) <Joao Duarte>
```
Comment (jsvd) [inline docs/static/releasenotes.asciidoc]: We can remove all of these beneath the TBD

**Pull request closed without merging**